JavaErickHernandez/Day3/Lists.java

```diff
@@ -0,0 +1,116 @@
+package bootcamp;
+import java.util.*;;
+
+public class Lists {
+	static List<Integer> numbers = new ArrayList<>();
+	static String numberOfInputs = "Enter the size of the list: ";
```

Comment:
What is the reason for not putting these variables as "final"? if you put "final" you avoid to any other person can change this variables in other par of the code.